Microsoft/WebTemplateStudio
# Get Pages through Extension instead of CoreTS directly

Task: https://microsoftgarage.visualstudio.com/Intern%20GitHub/_workitems/edit/30277

**Changes**
- Currently, the client directly calls the CoreTS Engine to get pages and put in the wizard Content reducer.
- Now, the client should communicate to the extension that it wants to get pages from CoreTS, extension makes request to the CoreTS and returns the response to the client, which then updates the state.

**How to Check**
- Launch the wizard
- Navigate through it - `Pages` page should be working as usual
- Try to generate a project - Project should be appropriately generated.
